pkg/cli/logfile/logfile_unix.go

```diff
@@ -33,7 +33,8 @@ func Hook() error {
 		return nil
 	}
 
-	isTerminal := term.IsTerminal(int(os.Stdout.Fd()))
+	// if both stdin and stdout are not terminals, then we don't need a pty
```

Comment:
Why do we need a terminal _at all_? I can't remember this.

Reply:
If we pipe to a file, terminal checks will fail. This means that logs will never be formatted. We use a pty to ensure that users can still use things that need user input and terminal drawing (e.g., `devenv top`) and have nicely formatted logs.